Find a functionally equivalent code:android.media.ImageReader.getNumPlanesFromFormat:COMMENT
Method Modifier: private     
Comment:/**
 * Only a subset of the formats defined in
 * {@link android.graphics.ImageFormat ImageFormat} and
 * {@link android.graphics.PixelFormat PixelFormat} are supported by
 * ImageReader. When reading RGB data from a surface, the formats defined in
 * {@link android.graphics.PixelFormat PixelFormat} can be used, when
 * reading YUV, JPEG or raw sensor data (for example, from camera or video
 * decoder), formats from {@link android.graphics.ImageFormat ImageFormat}
 * are used.
 */

@@ -15,20 +15,21 @@
         case ImageFormat.Y8:
         case ImageFormat.Y16:
         case ImageFormat.RAW_SENSOR:
+        case ImageFormat.RAW10:
             return 1;
         default:
             throw new UnsupportedOperationException(String.format("Invalid format specified %d", mFormat));

